docs/content/documentation/systems/Kernels/tensor_mechanics/InertialForceBeam.md

```diff
@@ -0,0 +1,8 @@
+# InertialForceBeam
```

Comment:
These files all need some descriptions of what the classes do.  These can be really short, and should all reference the very thorough documentation you provided in C0TimoshenkoBeam.md.